### PR TITLE
Adds an option for referencing online resources

### PR DESCRIPTION
**Pull request by stansidel**
Hi! I've added an option to format references differently according to the item type. Specifically, there's a separate composition of the `@online` references, like [Wolfram](http://tex.stackexchange.com/a/51747).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -85,7 +77,44 @@ module.exports = {
     }
 };
 
+function valuesForKeys(keys, dict) {
```
Comment (canadaduane) [inline index.js:80]: It looks like there are some spacing issues in this function--we're using 4-spaces per indent level I believe.

**Review by canadaduane — COMMENTED** [review]
Looks good, thanks! Just some formatting issues.

File: index.js (hunk 1 above)
Comment (stansidel) [reply index.js:80]: Sorry for this chunk. I just pasted it from the online editor and didn't notice the issue. Fixed that.

**Comment by canadaduane** [comment]
Thanks!

**Pull request merged**